### DISCUSSION TITLE
How to configure for multiple windows?

### DISCUSSION THREAD

**Discussion by crskycode** [alex8088/electron-vite#618] [Q&A]
I have the following folder structure, which contains an app window and an about window.

![image](https://github.com/user-attachments/assets/0ec4474c-fda2-4590-a540-012701a34ce9)

How should I write the configuration file?

![image](https://github.com/user-attachments/assets/686da27c-6e00-4c37-a13a-c453a2d65fab)

**Comment by alex8088** [comment]
https://electron-vite.org/guide/dev#multiple-windows-app
